src/He.PipelineAssessment.UI.Tests/Features/Admin/AssessmentToolManagement/Validators/CreateAssessmentToolWorkflowCommandValidatorTests.cs

```diff
@@ -35,5 +45,34 @@ public void MultipleValidator_Should_ReturnValidationMessage_WhenPropertyIsInval
             Assert.NotEqual("The Workflow Definition Id can be empty.",
```

Comment:
I think it would be better if we test each part of the validator separately, checking the relevant message is either there or not there. I don't think we are fully testing that no validation message appears when the properties pass the validation.